### PR TITLE
 Fix flaky testing

### LINKED ISSUE #1951: Tests are flaky and may fail randomly on Azure Pipelines
Sometimes icon tests - in particular those involving long knobs URLs - fail.

Not sure if this is connected to maybe the storybook update or the adding of materials, but it could be that the icon page takes too long to load and therefore the tests fail on Azure. 

But investigation is needed here.

<del>Would be cool if we could just migrate to playwright already.</del>

The issue is broader and sometimes tests also fail locally on MacBook, and they may fail on other tests than just certain icon tests.

Systematically investigate and find reasons before implementing principled solutions.

### PR DESCRIPTION
**Pull request by markus-walther**
Fixes #1951

# <del>Things to try next:</del>
- [ ] detect that we are on Azure CI (how?) and reduce concurrency (Testcafe -c), assuming ressource overloading as cause
- [ ] get beefier (paid?) machine on Azure CI and check whether problem(s) go away
- [ ] Try other free-for-public-github-repos CI offerings (esp. CircleCI)

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.

### CODE REVIEW

File: scripts/testcafe-chrome.sh [hunk 1]
```diff
@@ -10,9 +10,9 @@ npx wait-on $TEST_HOST_STORYBOOK_URL -t 30000
 
 if [ -z "$1" ]
 then
-    npx testcafe -c ${NUM_CPUS} "chrome:headless" -c 4 ./**/ui.test.js
+    npx testcafe -c ${NUM_CPUS} "chrome:headless" -q ./**/ui.test.js
 else
-    npx testcafe -F "${1}" -c ${NUM_CPUS} "chrome:headless" -c 4 ./**/ui.test.js
+    npx testcafe -F "${1}" -c ${NUM_CPUS} "chrome:headless" -q ./**/ui.test.js
```
Comment (danyball) [inline scripts/testcafe-chrome.sh:15]: Nice! https://devexpress.github.io/testcafe/documentation/guides/basic-guides/run-tests.html#quarantine-mode

File: src/components/10-atoms/link/ui.test.js [hunk 2]
```diff
@@ -357,13 +357,13 @@ test('should correctly update icon and text', async t => {
     .expect(await iconSVGPathElement.getAttribute('d'))
     .eql(downIconSignature);
 
-  await t.expect(await axaLink.textContent).eql('Mehr Filter');
+  await t.expect(axaLink.textContent).eql('Mehr Filter');
```
Comment (danyball) [inline src/components/10-atoms/link/ui.test.js:360]: Too many `await` can cause a delay?

**Review by danyball — DISMISSED** [review]

File: src/components/10-atoms/link/ui.test.js (hunk 2 above)
Comment (markus-walther) [reply src/components/10-atoms/link/ui.test.js:360]: The new version of Testcafe spit out lots of warnings about using them the wrong way - I fixed all warnings.

File: scripts/testcafe-chrome.sh (hunk 1 above)
Comment (markus-walther) [reply scripts/testcafe-chrome.sh:15]: Thanks

**Review by raphaellueckl — DISMISSED** [review]
very nice improvements!

**Review by danyball — APPROVED** [review]

**Review by raphaellueckl — APPROVED** [review]

**Pull request merged**